[REMOVED]

yf225 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#29633 Support C++ tensor advanced indexing**
* #29689 Add more tests for torch::arange
* #29632 Use default dtype for torch::tensor(floating_point_values) and torch::tensor(empty braced-init-list) when dtype is not specified

Commit:
Support C++ tensor advanced indexing

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 29633 gh/yf225/67/head

kostmo commented:
## CircleCI build failures summary
As of commit 5873406:
* **1/1** failures introduced in this PR
* **0/1** recognized as flaky



Here are the [reasons each build failed](https://dr.pytorch.org/commit-details.html?sha1=5873406eb10d50635848eca699e40d76749a0ccd).

---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 29633 gh/yf225/67/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 29633 gh/yf225/67/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 29633 gh/yf225/67/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 29633 gh/yf225/67/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 29633 gh/yf225/67/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 29633 gh/yf225/67/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 29633 gh/yf225/67/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 29633 gh/yf225/67/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 29633 gh/yf225/67/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 29633 gh/yf225/67/head
